Point.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Point.h"
+#include <cmath>
+
+ /* ----------------------------------------------------------------------------
+   * Copyright &copy; 2015 Randy Tanizawa <rtanizawa@csu.fullerton.edu>
+   * Released under the [MIT License] (http://opensource.org/licenses/MIT)
+   * ------------------------------------------------------------------------- */
+
+  /**
+   * Method prototypes for class Point.
+   */
+
+Point::Point()
+{
+    x = 0;
+    y = 0;
+}
+
+Point::Point(double X, double Y)
```

Comment:
dangerous to have the difference between two variables be a matter of capitalization -- and definitely unusual to have variables starting with a capital letter (unless they're macros or constants, and all caps).  though in this case it's clear what you're doing